packages/docker/src/generators/init/init.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  type Tree,
+  type GeneratorCallback,
+  readNxJson,
+  createProjectGraphAsync,
+  addDependenciesToPackageJson,
+  formatFiles,
+  runTasksInSerial,
+  logger,
+} from '@nx/devkit';
+import { addPlugin } from '@nx/devkit/src/utils/add-plugin';
+import { InitGeneratorSchema } from './schema';
+import { createNodesV2 } from '../../plugins/plugin';
+import { nxVersion } from '../../utils/versions';
+
+export function updateDependencies(tree: Tree, schema: InitGeneratorSchema) {
+  return addDependenciesToPackageJson(
+    tree,
+    {},
+    {
+      '@nx/docker': nxVersion,
+    },
+    undefined,
+    schema.keepExistingVersions
+  );
+}
+
+export function initGenerator(tree: Tree, schema: InitGeneratorSchema) {
+  return initGeneratorInternal(tree, { addPlugin: false, ...schema });
+}
+
+export async function initGeneratorInternal(
+  tree: Tree,
+  schema: InitGeneratorSchema
+) {
+  logger.warn(
+    `Docker support is experimental. Breaking changes may occur and not adhere to semver versioning.`
+  );
+  const nxJson = readNxJson(tree);
+  const addPluginDefault =
```

Comment:
Please remove the option to not add the plugin... it would be very hard to configure without it.